### DISCUSSION TITLE
Please can I have some help

### DISCUSSION THREAD

**Discussion by jnchay548** [wills106/homeassistant-solax-modbus#345] [General]
Hi there,

THANKS SO MUCH FOR THE COMPONENT.

I have the modus interface working and I am able to change my timer etc via HA - GREAT!!!

I have a Solis 3.0kW 5G RAI Energy Storage AC Coupled Battery Charger

But I am stuck trying to figure out the simplest way to answer the following question:

I would like to be able to charge my home battery with some defined periods -

2330 -- 0530

AND

If octopus intelligent issues another cheap period AND the car is plugged in and charging during this period.

I am very naiive when it comes to yaml etc so go easy on me.

Is this doable?

Thanks!

**Comment by alienatedsec** [comment]
It is doable, but you will need to learn more than you already committed. Check[ those Wiki pages](https://github.com/wills106/homeassistant-solax-modbus/wiki/Modbus-Power-Control-(remotecontrol-power)) and you should be fine
